src/js/calculator.js

```diff
@@ -0,0 +1,74 @@
+import { isNumber } from './utils.js';
+import { DEFAULT_NUMBER, OPERATOR } from './const.js';
+
+const OPERRATIONS = {
+  [OPERATOR.SUM]: (num1, num2) => num1 + num2,
+  [OPERATOR.SUBSTRACT]: (num1, num2) => num1 - num2,
+  [OPERATOR.MULTIPLY]: (num1, num2) => num1 * num2,
+  [OPERATOR.DIVIDE]: (num1, num2) => Math.floor(num1 / num2),
+};
+
+class Calculator {
+  #num1;
+  #num2;
+  #output;
+  #operator;
+
+  constructor() {
+    this.#num1 = null;
+    this.#num2 = null;
+    this.#output = null;
+    this.#operator = null;
+  }
+
+  setNumber = (nextNumber) => {
+    if (this.#operator) {
+      this.#num2 = nextNumber;
+    } else {
+      this.#num1 = nextNumber;
+    }
+  };
+
+  setOperator = (nextOperator) => {
+    this.#operator = nextOperator;
+  };
+
+  clear = () => {
+    this.#num1 = null;
+    this.#num2 = null;
+    this.#output = null;
+    this.#operator = null;
+  };
+
+  compute = () => {
+    this.#output = OPERRATIONS[this.#operator](this.#num1, this.#num2);
```

Comment:
```suggestion
    const operation = OPERRATIONS[this.#operator];
    this.#output = operation(this.#num1, this.#num2);
```
개인적으로는 이렇게 함수에 네이밍을 붙혀서 재선언하고 사용하는게 더 이해하기 쉽더라구요 🙂

Reply:
짧게 작성한다고 좋은 코드가 아닌거 같네요 ㅎㅎ 제안해주신것처럼 재선언하는 방식이 더욱 읽기 편해진것 같아요 ! !